src/domains/todo/screens/AsyncLocalTodoScreen.tsx

```diff
@@ -4,7 +4,8 @@ import { Button } from 'src/components/Button';
 import { Screen } from 'src/components/Screen';
 import { Text } from 'src/components/Text';
 import { Todo } from 'src/domains/todo/components/Todo';
-import { createTodo, listTodos, Todo as TodoType } from 'src/services/todoApi';
+import { createTodo, listTodos } from 'src/services/todo/api';
+import { Todo as TodoType } from 'src/services/todo/types';
```

Comment:
Could add an index.ts similar to exampleService's that exports everything, so this import statement becomes:

```suggestion
import { createTodo, listTodos, Todo as TodoType } from 'src/services/todo';
```